presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -102,8 +102,67 @@ Property Name                                      Description
 ``hive.immutable-partitions``                      Can new data be inserted into existing partitions?           ``false``
 
 ``hive.max-partitions-per-writers``                Maximum number of partitions per writer.                     100
+================================================== ============================================================ ==========
+
+Amazon S3 Configuration
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The Hive connector also allows querying data in Hive where data files are stored in Amazon S3. 
+The SQL support for S3 tables remains the same as for HDFS tables.
+
+For accessing tables stored in S3 the mandatory configuration properties required
+are the AWS credential properties ``hive.s3.aws-access-key`` and ``hive.s3.aws-secret-key``. 
+You can also use ``hive.s3.use-instance-credentials`` which if set to true, enables retrieving
```

Comment:
This is added instead of default credentials. I have that info in the table below. Actually do you think we need to add all this info about use-instance-credentials property here? I think the users familiar with aws will know about this. Maybe I can just remove this line.